lionblog-eunjae/src/components/Posts/index.jsx

```diff
@@ -15,4 +19,31 @@ export const SmallPost = ({ post }) => {
       </div>
     </div>
   );
+};
+
+
+export const BigPost = ({ post }) => {
+  return (
+    <div className="bg-orange-400 w-[50vw] h-[30vh] mt-7 mb-5 rounded-xl border-orange-300 border-4">
+      <div className="text-black flex flex-col m-6">
+        <h3 className="text-xl mb-4 font-bold">{`${post.author.username}의 ${post.title}`}</h3>
+        <p className="border-black border-2 rounded-xl p-2 text-lg mb-4">
+          {post.content}
+        </p>
+        <div className="flex flex-wrap mt-5">
+          {post.tags.map((tag) => (
```

Comment:
```suggestion
          {post.tags &&
          post.tags.map((tag) => (
            <span key={tag.id} className="tag">
              #{tag.content}
            </span>
          ))}
```
이런 느낌으로 post.tags가 없을 때 관련한 예외 처리도 해주면 에러 발생 가능성이 줄 것 같아요!